commands/inventory/add_node_remote.go

```diff
@@ -0,0 +1,96 @@
+// pmm-admin
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package inventory
+
+import (
+	"github.com/percona/pmm/api/inventorypb/json/client"
+	"github.com/percona/pmm/api/inventorypb/json/client/nodes"
+
+	"github.com/percona/pmm-admin/commands"
+)
+
+var addNodeRemoteResultT = commands.ParseTemplate(`
+Remote Node added.
+Node ID  : {{ .Node.NodeID }}
+Node name: {{ .Node.NodeName }}
+
+Address       : {{ .Node.Address }}
+Custom labels : {{ .Node.CustomLabels }}
+
+Region    : {{ .Node.Region }}
+Az        : {{ .Node.Az }}
+`)
+
+type addNodeRemoteResult struct {
+	Node *nodes.AddRemoteNodeOKBodyRemote `json:"Remote"`
```

Comment:
All our JSON tags should be in `lower_snake_case`. Change to `"remote"` there, and check everywhere.